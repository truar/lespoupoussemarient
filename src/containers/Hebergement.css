.Hebergement {
    background: url('../images/hebergement.jpg') no-repeat center fixed;
    background-size: cover; /* version standardis√©e */
    height: 100vh;
}

.Hebergement .wrapper-title h1 {
    font-size: 70px;
    font-family: SignPainter;
    color: black;
    margin-block-start: 0px;
}
.Hebergement .wrapper-title {
    padding-top: 10px;
    z-index: 100;
    text-align: center;
    width: 100%;
}

.Hebergement article {
    width: 90%;
    display: flex;
    flex-direction: column;
    align-items: flex-start;
    background-color: rgba(37,71,110, 0.4);
    border: 1px solid white;
    border-radius: 2px;
    padding: 5px 10px;
}

.Hebergement article .content {
    width: 100%;
    display: flex;
    flex-direction: column;
}

.Hebergement .WrapperContent {
    flex-grow: 2;
}

.Hebergement h2 {
    font-size: 1.7rem;
}

@media (min-width: 768px) {
    .Hebergement article {
        width: 80%;
        flex-grow: 0.1;
        justify-content: space-evenly;
    }

    .Hebergement article .content {
        flex-direction: row;
        justify-content: space-between;
    }
    
}

@media (min-width: 992px) {

    .Hebergement h2 {
        font-size: 3rem;
    }
}

