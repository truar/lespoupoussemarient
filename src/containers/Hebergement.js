import React, { Component } from 'react';
import PageTitle from '../components/PageTitle';
import Section from '../components/Section';
import WrapperContent from '../components/WrapperContent';
import './Hebergement.css';
import pdf from '../files/hebergements.pdf'

const lodgingTypes = [
    {
        id: 'chalet',
        title: 'Chalets',
        infos: [[
            'Sur place',
            'Prix pour la nuit : 35€/pers'
        ], [
            'Place limitées',
            'Capacité : 7 personnes'
        ]]
    }, {
        id: 'tente',
        title: 'Tentes*',
        infos: [[
            'Sur place',
            'Prix pour la nuit : gratuit'
        ], [
            'Place limitées',
            '*matériel non fourni par les mariés'
        ]]
    }, {
        id: 'gite',
        title: 'Gites*',
        infos: [[
            'Aux alentours',
            'Prix pour la nuit : 40€/pers'
        ], [
            'A réserver',
            <a href={pdf} download>Informations hébergement</a>
        ]]
    }
]

class Hebergement extends Component {

    render() {
        const contents = lodgingTypes.map((lodgingType, i) => {
            const infos = lodgingType.infos.map((info, i) => {
                const lines = info.map((line, i) => {
                    return (
                        <p key={i}>{line}</p>
                    );
                });
                return (
                    <div key={i} className="content">
                        {lines}
                    </div>
                );
            });
            return (
                <article key={i} id={lodgingType.id}>
                    <h2>{lodgingType.title}</h2>
                    {infos}
                </article>
            )
        });

        return (
            <Section id="Hebergement" className="Hebergement" {...this.props}>
                <PageTitle title="Hébergement" />
                <WrapperContent>
                    {contents}
                </WrapperContent>
            </Section>
        );
    }
}

export default Hebergement;