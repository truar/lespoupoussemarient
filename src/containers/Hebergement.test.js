import React from 'react';
import Hebergement from './Hebergement';
import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const component = shallow(<Hebergement />);
  expect(component).toMatchSnapshot();
});
