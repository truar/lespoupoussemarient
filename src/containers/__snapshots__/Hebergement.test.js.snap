// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Hebergement />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <PageTitle
          title="Hébergement"
        />,
        <WrapperContent>
          <article
            id="chalet"
          >
            <h2>
              Chalets
            </h2>
            <div
              className="content"
            >
              <p>
                Sur place
              </p>
              <p>
                Prix pour la nuit : 40€/pers
              </p>
            </div>
            <div
              className="content"
            >
              <p>
                Place limitées
              </p>
              <p>
                Capacité : 7 personnes
              </p>
            </div>
          </article>
          <article
            id="tente"
          >
            <h2>
              Tentes*
            </h2>
            <div
              className="content"
            >
              <p>
                Sur place
              </p>
              <p>
                Prix pour la nuit : gratuit
              </p>
            </div>
            <div
              className="content"
            >
              <p>
                Place limitées
              </p>
              <p>
                *matériel non fourni par les mariés
              </p>
            </div>
          </article>
          <article
            id="gite"
          >
            <h2>
              Gites
            </h2>
            <div
              className="content"
            >
              <p>
                Aux alentours
              </p>
              <p>
                Prix pour la nuit : 40€/pers
              </p>
            </div>
            <div
              className="content"
            >
              <p>
                A réserver
              </p>
              <p>
                <a
                  download={true}
                  href="hebergements.pdf"
                >
                  Informations hébergement (pdf)
                </a>
              </p>
            </div>
          </article>
        </WrapperContent>,
      ],
      "className": "Hebergement",
      "id": "Hebergement",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "title": "Hébergement",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <article
              id="chalet"
            >
              <h2>
                Chalets
              </h2>
              <div
                className="content"
              >
                <p>
                  Sur place
                </p>
                <p>
                  Prix pour la nuit : 40€/pers
                </p>
              </div>
              <div
                className="content"
              >
                <p>
                  Place limitées
                </p>
                <p>
                  Capacité : 7 personnes
                </p>
              </div>
            </article>,
            <article
              id="tente"
            >
              <h2>
                Tentes*
              </h2>
              <div
                className="content"
              >
                <p>
                  Sur place
                </p>
                <p>
                  Prix pour la nuit : gratuit
                </p>
              </div>
              <div
                className="content"
              >
                <p>
                  Place limitées
                </p>
                <p>
                  *matériel non fourni par les mariés
                </p>
              </div>
            </article>,
            <article
              id="gite"
            >
              <h2>
                Gites
              </h2>
              <div
                className="content"
              >
                <p>
                  Aux alentours
                </p>
                <p>
                  Prix pour la nuit : 40€/pers
                </p>
              </div>
              <div
                className="content"
              >
                <p>
                  A réserver
                </p>
                <p>
                  <a
                    download={true}
                    href="hebergements.pdf"
                  >
                    Informations hébergement (pdf)
                  </a>
                </p>
              </div>
            </article>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "0",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h2>
                  Chalets
                </h2>,
                Array [
                  <div
                    className="content"
                  >
                    <p>
                      Sur place
                    </p>
                    <p>
                      Prix pour la nuit : 40€/pers
                    </p>
                  </div>,
                  <div
                    className="content"
                  >
                    <p>
                      Place limitées
                    </p>
                    <p>
                      Capacité : 7 personnes
                    </p>
                  </div>,
                ],
              ],
              "id": "chalet",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Chalets",
                },
                "ref": null,
                "rendered": "Chalets",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": "0",
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <p>
                      Sur place
                    </p>,
                    <p>
                      Prix pour la nuit : 40€/pers
                    </p>,
                  ],
                  "className": "content",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": "0",
                    "nodeType": "host",
                    "props": Object {
                      "children": "Sur place",
                    },
                    "ref": null,
                    "rendered": "Sur place",
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": "1",
                    "nodeType": "host",
                    "props": Object {
                      "children": "Prix pour la nuit : 40€/pers",
                    },
                    "ref": null,
                    "rendered": "Prix pour la nuit : 40€/pers",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              Object {
                "instance": null,
                "key": "1",
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <p>
                      Place limitées
                    </p>,
                    <p>
                      Capacité : 7 personnes
                    </p>,
                  ],
                  "className": "content",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": "0",
                    "nodeType": "host",
                    "props": Object {
                      "children": "Place limitées",
                    },
                    "ref": null,
                    "rendered": "Place limitées",
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": "1",
                    "nodeType": "host",
                    "props": Object {
                      "children": "Capacité : 7 personnes",
                    },
                    "ref": null,
                    "rendered": "Capacité : 7 personnes",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
            ],
            "type": "article",
          },
          Object {
            "instance": null,
            "key": "1",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h2>
                  Tentes*
                </h2>,
                Array [
                  <div
                    className="content"
                  >
                    <p>
                      Sur place
                    </p>
                    <p>
                      Prix pour la nuit : gratuit
                    </p>
                  </div>,
                  <div
                    className="content"
                  >
                    <p>
                      Place limitées
                    </p>
                    <p>
                      *matériel non fourni par les mariés
                    </p>
                  </div>,
                ],
              ],
              "id": "tente",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Tentes*",
                },
                "ref": null,
                "rendered": "Tentes*",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": "0",
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <p>
                      Sur place
                    </p>,
                    <p>
                      Prix pour la nuit : gratuit
                    </p>,
                  ],
                  "className": "content",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": "0",
                    "nodeType": "host",
                    "props": Object {
                      "children": "Sur place",
                    },
                    "ref": null,
                    "rendered": "Sur place",
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": "1",
                    "nodeType": "host",
                    "props": Object {
                      "children": "Prix pour la nuit : gratuit",
                    },
                    "ref": null,
                    "rendered": "Prix pour la nuit : gratuit",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              Object {
                "instance": null,
                "key": "1",
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <p>
                      Place limitées
                    </p>,
                    <p>
                      *matériel non fourni par les mariés
                    </p>,
                  ],
                  "className": "content",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": "0",
                    "nodeType": "host",
                    "props": Object {
                      "children": "Place limitées",
                    },
                    "ref": null,
                    "rendered": "Place limitées",
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": "1",
                    "nodeType": "host",
                    "props": Object {
                      "children": "*matériel non fourni par les mariés",
                    },
                    "ref": null,
                    "rendered": "*matériel non fourni par les mariés",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
            ],
            "type": "article",
          },
          Object {
            "instance": null,
            "key": "2",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h2>
                  Gites
                </h2>,
                Array [
                  <div
                    className="content"
                  >
                    <p>
                      Aux alentours
                    </p>
                    <p>
                      Prix pour la nuit : 40€/pers
                    </p>
                  </div>,
                  <div
                    className="content"
                  >
                    <p>
                      A réserver
                    </p>
                    <p>
                      <a
                        download={true}
                        href="hebergements.pdf"
                      >
                        Informations hébergement (pdf)
                      </a>
                    </p>
                  </div>,
                ],
              ],
              "id": "gite",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Gites",
                },
                "ref": null,
                "rendered": "Gites",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": "0",
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <p>
                      Aux alentours
                    </p>,
                    <p>
                      Prix pour la nuit : 40€/pers
                    </p>,
                  ],
                  "className": "content",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": "0",
                    "nodeType": "host",
                    "props": Object {
                      "children": "Aux alentours",
                    },
                    "ref": null,
                    "rendered": "Aux alentours",
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": "1",
                    "nodeType": "host",
                    "props": Object {
                      "children": "Prix pour la nuit : 40€/pers",
                    },
                    "ref": null,
                    "rendered": "Prix pour la nuit : 40€/pers",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              Object {
                "instance": null,
                "key": "1",
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <p>
                      A réserver
                    </p>,
                    <p>
                      <a
                        download={true}
                        href="hebergements.pdf"
                      >
                        Informations hébergement (pdf)
                      </a>
                    </p>,
                  ],
                  "className": "content",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": "0",
                    "nodeType": "host",
                    "props": Object {
                      "children": "A réserver",
                    },
                    "ref": null,
                    "rendered": "A réserver",
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": "1",
                    "nodeType": "host",
                    "props": Object {
                      "children": <a
                        download={true}
                        href="hebergements.pdf"
                      >
                        Informations hébergement (pdf)
                      </a>,
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Informations hébergement (pdf)",
                        "download": true,
                        "href": "hebergements.pdf",
                      },
                      "ref": null,
                      "rendered": "Informations hébergement (pdf)",
                      "type": "a",
                    },
                    "type": "p",
                  },
                ],
                "type": "div",
              },
            ],
            "type": "article",
          },
        ],
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <PageTitle
            title="Hébergement"
          />,
          <WrapperContent>
            <article
              id="chalet"
            >
              <h2>
                Chalets
              </h2>
              <div
                className="content"
              >
                <p>
                  Sur place
                </p>
                <p>
                  Prix pour la nuit : 40€/pers
                </p>
              </div>
              <div
                className="content"
              >
                <p>
                  Place limitées
                </p>
                <p>
                  Capacité : 7 personnes
                </p>
              </div>
            </article>
            <article
              id="tente"
            >
              <h2>
                Tentes*
              </h2>
              <div
                className="content"
              >
                <p>
                  Sur place
                </p>
                <p>
                  Prix pour la nuit : gratuit
                </p>
              </div>
              <div
                className="content"
              >
                <p>
                  Place limitées
                </p>
                <p>
                  *matériel non fourni par les mariés
                </p>
              </div>
            </article>
            <article
              id="gite"
            >
              <h2>
                Gites
              </h2>
              <div
                className="content"
              >
                <p>
                  Aux alentours
                </p>
                <p>
                  Prix pour la nuit : 40€/pers
                </p>
              </div>
              <div
                className="content"
              >
                <p>
                  A réserver
                </p>
                <p>
                  <a
                    download={true}
                    href="hebergements.pdf"
                  >
                    Informations hébergement (pdf)
                  </a>
                </p>
              </div>
            </article>
          </WrapperContent>,
        ],
        "className": "Hebergement",
        "id": "Hebergement",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "title": "Hébergement",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <article
                id="chalet"
              >
                <h2>
                  Chalets
                </h2>
                <div
                  className="content"
                >
                  <p>
                    Sur place
                  </p>
                  <p>
                    Prix pour la nuit : 40€/pers
                  </p>
                </div>
                <div
                  className="content"
                >
                  <p>
                    Place limitées
                  </p>
                  <p>
                    Capacité : 7 personnes
                  </p>
                </div>
              </article>,
              <article
                id="tente"
              >
                <h2>
                  Tentes*
                </h2>
                <div
                  className="content"
                >
                  <p>
                    Sur place
                  </p>
                  <p>
                    Prix pour la nuit : gratuit
                  </p>
                </div>
                <div
                  className="content"
                >
                  <p>
                    Place limitées
                  </p>
                  <p>
                    *matériel non fourni par les mariés
                  </p>
                </div>
              </article>,
              <article
                id="gite"
              >
                <h2>
                  Gites
                </h2>
                <div
                  className="content"
                >
                  <p>
                    Aux alentours
                  </p>
                  <p>
                    Prix pour la nuit : 40€/pers
                  </p>
                </div>
                <div
                  className="content"
                >
                  <p>
                    A réserver
                  </p>
                  <p>
                    <a
                      download={true}
                      href="hebergements.pdf"
                    >
                      Informations hébergement (pdf)
                    </a>
                  </p>
                </div>
              </article>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": "0",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2>
                    Chalets
                  </h2>,
                  Array [
                    <div
                      className="content"
                    >
                      <p>
                        Sur place
                      </p>
                      <p>
                        Prix pour la nuit : 40€/pers
                      </p>
                    </div>,
                    <div
                      className="content"
                    >
                      <p>
                        Place limitées
                      </p>
                      <p>
                        Capacité : 7 personnes
                      </p>
                    </div>,
                  ],
                ],
                "id": "chalet",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Chalets",
                  },
                  "ref": null,
                  "rendered": "Chalets",
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": "0",
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <p>
                        Sur place
                      </p>,
                      <p>
                        Prix pour la nuit : 40€/pers
                      </p>,
                    ],
                    "className": "content",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": "0",
                      "nodeType": "host",
                      "props": Object {
                        "children": "Sur place",
                      },
                      "ref": null,
                      "rendered": "Sur place",
                      "type": "p",
                    },
                    Object {
                      "instance": null,
                      "key": "1",
                      "nodeType": "host",
                      "props": Object {
                        "children": "Prix pour la nuit : 40€/pers",
                      },
                      "ref": null,
                      "rendered": "Prix pour la nuit : 40€/pers",
                      "type": "p",
                    },
                  ],
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": "1",
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <p>
                        Place limitées
                      </p>,
                      <p>
                        Capacité : 7 personnes
                      </p>,
                    ],
                    "className": "content",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": "0",
                      "nodeType": "host",
                      "props": Object {
                        "children": "Place limitées",
                      },
                      "ref": null,
                      "rendered": "Place limitées",
                      "type": "p",
                    },
                    Object {
                      "instance": null,
                      "key": "1",
                      "nodeType": "host",
                      "props": Object {
                        "children": "Capacité : 7 personnes",
                      },
                      "ref": null,
                      "rendered": "Capacité : 7 personnes",
                      "type": "p",
                    },
                  ],
                  "type": "div",
                },
              ],
              "type": "article",
            },
            Object {
              "instance": null,
              "key": "1",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2>
                    Tentes*
                  </h2>,
                  Array [
                    <div
                      className="content"
                    >
                      <p>
                        Sur place
                      </p>
                      <p>
                        Prix pour la nuit : gratuit
                      </p>
                    </div>,
                    <div
                      className="content"
                    >
                      <p>
                        Place limitées
                      </p>
                      <p>
                        *matériel non fourni par les mariés
                      </p>
                    </div>,
                  ],
                ],
                "id": "tente",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Tentes*",
                  },
                  "ref": null,
                  "rendered": "Tentes*",
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": "0",
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <p>
                        Sur place
                      </p>,
                      <p>
                        Prix pour la nuit : gratuit
                      </p>,
                    ],
                    "className": "content",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": "0",
                      "nodeType": "host",
                      "props": Object {
                        "children": "Sur place",
                      },
                      "ref": null,
                      "rendered": "Sur place",
                      "type": "p",
                    },
                    Object {
                      "instance": null,
                      "key": "1",
                      "nodeType": "host",
                      "props": Object {
                        "children": "Prix pour la nuit : gratuit",
                      },
                      "ref": null,
                      "rendered": "Prix pour la nuit : gratuit",
                      "type": "p",
                    },
                  ],
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": "1",
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <p>
                        Place limitées
                      </p>,
                      <p>
                        *matériel non fourni par les mariés
                      </p>,
                    ],
                    "className": "content",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": "0",
                      "nodeType": "host",
                      "props": Object {
                        "children": "Place limitées",
                      },
                      "ref": null,
                      "rendered": "Place limitées",
                      "type": "p",
                    },
                    Object {
                      "instance": null,
                      "key": "1",
                      "nodeType": "host",
                      "props": Object {
                        "children": "*matériel non fourni par les mariés",
                      },
                      "ref": null,
                      "rendered": "*matériel non fourni par les mariés",
                      "type": "p",
                    },
                  ],
                  "type": "div",
                },
              ],
              "type": "article",
            },
            Object {
              "instance": null,
              "key": "2",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2>
                    Gites
                  </h2>,
                  Array [
                    <div
                      className="content"
                    >
                      <p>
                        Aux alentours
                      </p>
                      <p>
                        Prix pour la nuit : 40€/pers
                      </p>
                    </div>,
                    <div
                      className="content"
                    >
                      <p>
                        A réserver
                      </p>
                      <p>
                        <a
                          download={true}
                          href="hebergements.pdf"
                        >
                          Informations hébergement (pdf)
                        </a>
                      </p>
                    </div>,
                  ],
                ],
                "id": "gite",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Gites",
                  },
                  "ref": null,
                  "rendered": "Gites",
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": "0",
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <p>
                        Aux alentours
                      </p>,
                      <p>
                        Prix pour la nuit : 40€/pers
                      </p>,
                    ],
                    "className": "content",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": "0",
                      "nodeType": "host",
                      "props": Object {
                        "children": "Aux alentours",
                      },
                      "ref": null,
                      "rendered": "Aux alentours",
                      "type": "p",
                    },
                    Object {
                      "instance": null,
                      "key": "1",
                      "nodeType": "host",
                      "props": Object {
                        "children": "Prix pour la nuit : 40€/pers",
                      },
                      "ref": null,
                      "rendered": "Prix pour la nuit : 40€/pers",
                      "type": "p",
                    },
                  ],
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": "1",
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <p>
                        A réserver
                      </p>,
                      <p>
                        <a
                          download={true}
                          href="hebergements.pdf"
                        >
                          Informations hébergement (pdf)
                        </a>
                      </p>,
                    ],
                    "className": "content",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": "0",
                      "nodeType": "host",
                      "props": Object {
                        "children": "A réserver",
                      },
                      "ref": null,
                      "rendered": "A réserver",
                      "type": "p",
                    },
                    Object {
                      "instance": null,
                      "key": "1",
                      "nodeType": "host",
                      "props": Object {
                        "children": <a
                          download={true}
                          href="hebergements.pdf"
                        >
                          Informations hébergement (pdf)
                        </a>,
                      },
                      "ref": null,
                      "rendered": Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "Informations hébergement (pdf)",
                          "download": true,
                          "href": "hebergements.pdf",
                        },
                        "ref": null,
                        "rendered": "Informations hébergement (pdf)",
                        "type": "a",
                      },
                      "type": "p",
                    },
                  ],
                  "type": "div",
                },
              ],
              "type": "article",
            },
          ],
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
